flowsa/data/flowbysectormethods/CNHWC_national_2014.yaml

```diff
@@ -0,0 +1,37 @@
+target_sector_level: NAICS_6
+target_sector_source: NAICS_2012_Code
+target_geoscale: national
+source_names:
+  "EPA_CDDPath":
+    data_format: 'FBA'
+    class: Other
+    geoscale_to_use: national
+    year: 2014
+    clean_fba_df_fxn: 'assign_wood_to_engineering'
+    mfl_mapping: 'EPA_CDDPath'
+    activity_sets:
+      activity_set_1:
+        names:
+        - "Buildings"
+        - "Other"
+        - "Other - Wood"
+        - "Roads and Bridges"
+        allocation_source: "Census_VIP"        
+        allocation_method: proportional
+        allocation_source_class: 'Money'
+        allocation_source_year: 2014
+        allocation_flow: 
+        - "Construction spending"
+        allocation_compartment: None
+        allocation_from_scale: national
+        allocation_helper: 'yes'
```

Comment:
@bl-young , no longer need "allocation_helper = yes", but having this parameter included won't cause any errors

Reply:
thanks, I had removed some but must have missed this one